tests/network/l2_bridge/migration_stuntime/conftest.py

```
@@ -0,0 +1,112 @@
+    ) as server_vm:
+        server_vm.start(wait=True)
+        server_vm.wait_for_agent_connected()
+        yield server_vm
+
+
+@pytest.fixture(scope="module")
+def stuntime_client_vm(
+    unprivileged_client: DynamicClient,
+    namespace: Namespace,
+    l2_bridge_stuntime_nad: libnad.NetworkAttachmentDefinition,
+    stuntime_server_vm: BaseVirtualMachine,
```

Ruff (0.15.14)

[warning] Unused function argument: `stuntime_server_vm`

(ARG001)